UPGRADING

```diff
@@ -1021,6 +1004,11 @@ PHP 8.4 UPGRADE NOTES
     effect, and is silently ignored. This underlying feature was
     deprecated in libcurl 7.11.1 and removed in 7.62.0.
 
+- GMP:
+  . Casting a GMP object to bool changed so that 0 becomes false and everything else
+    becomes true.
+    RFC: https://wiki.php.net/rfc/fix_up_bcmath_number_class
```

Comment:
Not sure about this one; *might* better be left in the incompatible section.

Reply:
The thing is that it wasn't possible to cast an instance of a GMP object to bool, it would trigger a `E_RECOVERABLE_ERROR` diagnostic.

Reply:
I should just reword it like I did in the migration guide PR.